layouts/partials/related-companies.html

```diff
@@ -9,8 +9,10 @@
         {{ $num_common_categories := intersect $.Params.categories .Params.categories | len }}
         {{ $num_common_countries := intersect ($.Param "relevant-countries") (.Param "relevant-countries") | len }}
         {{ if and (gt $num_common_categories 0) (ne .Permalink $.Permalink) }}
-            {{ $score := add (div (float $num_common_categories) $num_categories) (div $num_common_countries $num_countries) }}
-            {{ $related_pages = $related_pages | append (dict "page" . "score" $score) }}
+            {{if not .Params.nsfw}}
```

Comment:
I think it would be cleaner to add this to the outer `if`.

Reply:
sure, done.